[Issue #452] Improving documentation (#465)
openshift/cloud-credential-operator: docs/gcp_workload_identity.md

### Steps to install an OpenShift Cluster with Workload Identity

1. Set the variable `$RELEASE_IMAGE`

>    `$RELEASE_IMAGE` should be a sufficiently new OpenShift release image that you want to deploy in your cluster.

Reviewer: This is ambiguous. By `sufficiently new` means how new? We have [support matrix](https://github.com/openshift/cloud-credential-operator#support-matrix) for that. It would be great if we can mention the exact version here or some kind of reference to the support matrix

   It should be like: `RELEASE_IMAGE=quay.io/openshift-release-dev/ocp-release:${RHOCP_version}-${Arch}`

>    Where the `RHOCP_version` is something like: `4.10.0-fc.4`, `4.9.3` and the `Arch` is `x86_64`

Reviewer 2: ```suggestion
   Where the `RHOCP_version` is something like: `4.10.0-fc.4` or `4.9.3` and the `Arch` is `x86_64`
```

2. Extract the GCP Credentials Request objects from the above release image. You must use version 4.7 or newer of the `oc` CLI.
   ```
   mkdir credreqs ; oc adm release extract --cloud=gcp --credentials-requests $RELEASE_IMAGE --to=./credreqs
   ```

> 3. Extract the OpenShift install binary from the release image and ccoctl from the related image

Reviewer 2: ```suggestion
3. Extract the `openshift-install` and `ccoctl` binaries from the release image.
```

### Steps to install an OpenShift Cluster with Workload Identity

1. Set the variable `$RELEASE_IMAGE`

   `$RELEASE_IMAGE` should be a recent and supported  OpenShift release image that you want to deploy in your cluster.
   Please refer to the [support matrix](../README.md#support-matrix) for compatibilities.

>    It should be like: `RELEASE_IMAGE=quay.io/openshift-release-dev/ocp-release:${RHOCP_version}-${Arch}`
> 
>    Where the `RHOCP_version` is something like: `4.10.0-fc.4`s or `4.9.3` and the `Arch` is `x86_64`

Reviewer: ```suggestion
   A sample release image would be `RELEASE_IMAGE=quay.io/openshift-release-dev/ocp-release:${RHOCP_version}-${Arch}`

   Where `RHOCP_version` is the OpenShift version (e.g `4.10.0-fc.4`s or `4.9.3`) and the `Arch` is the architecture type (e.g `x86_64`) 
```
